config/chime_science_run_gpu.yaml

```diff
@@ -1017,6 +1019,7 @@ read_gain:
   updatable_config:
     gain_frb: /updatable_config/frb_gain
     gain_tracking: /updatable_config/tracking_gain
+    UT1_UTC: /updatable_config/UT1_UTC_offset
```

Comment:
We'll need to add the `FloatPhaseUpdate` stage as a new entry, then add to that entry the block:
```
updatable_config:
  UT1_UTC: /updatable_config/UT1_UTC_offset
```

Reply:
What should the name of this stage be? `float_phase_update` like the `read_gain` stage?

Reply:
I think `beamformer_phase_generation`.  Since that's basically what this stage is doing.

Reply:
Done!